source/_faq/component.markdown

```diff
@@ -4,6 +4,14 @@ description: "My integration does not show up"
 ha_category: Configuration
 ---
 
-When an integration does not show up, many different things can be the case. Before you try any of these steps, make sure to look at the `home-assistant.log` file and see if there are any errors related to your integration you are trying to set up.
+When an integration does not show up, many different things can be the case. Before you try any of these steps, make sure to: 
 
-If you have incorrect entries in your configuration files you can use the `check_config` script to assist in identifying them: `hass --script check_config`.
+- Restart the Home Assistant server
+  - From the web interface, go to `Configuration > Server Controls > Server Management` and click the "Restart" button
+- Look at the `home-assistant.log` file and see if there are any errors related to the integration you are trying to set up
```

Comment:
The logs can be viewed from the frontend. It might be friendlier to point to that first.